container.go

```diff
@@ -177,7 +178,7 @@ func (c *container) Delete(ctx context.Context, opts ...DeleteOpts) error {
 		return err
 	}
 	for _, o := range opts {
-		if err := o(ctx, c.client, r); err != nil {
+		if err := o(ctx, c.client, r); err != nil && !strings.Contains(err.Error(), fmt.Sprintf("snapshot %s does not exist", c.id)) {
```

Comment:
There are situations where the snapshot is not named after the container id.  This change looks good but we should change this so that we don't use the exact snapshot name